docs/develop/typescript/data-encryption.mdx

```diff
@@ -11,41 +11,129 @@ tags:
 description: Learn how to create a custom Payload Converter in TypeScript with Temporal SDKs to handle non-JSON-serializable values, configure your Data Converter, and utilize protobufs and encryption seamlessly in your Workflows and Activities.
 ---
 
-## How to use a custom payload converter in TypeScript {#custom-payload-conversion}
+## Payload Converter and Payload Codec Summary
 
-Temporal SDKs provide a [Payload Converter](/dataconversion#payload-converter) that can be customized to convert a custom data type to a [Payload](/dataconversion#payload) and back.
+This section will summarize the difference between a Payload Converter and Payload Codec.
```

Comment:
Avoid modal auxilliary (will)

"This section summarizes" presents the information as immediately relevant and actionable.

Reply:
I think it would help to add a summary to your intro to prep your readers for the main topic. For example: 

"Payloads are structured data transmitted between components in a Temporal installation. Sending payloads as plain text exposes security risks and increases bandwidth usage. Payload codecs and converters address these issues. Codecs provide a standardized format for data transmission, while converters handle the process of converting data into and out of that format. This page describes how to create custom Payload Converters in TypeScript to manage values that cannot be serialized with JSON. You'll configure a Data Converter for serialization and integrate Protobufs and encryption into your Workflows and Activities."